internal/controller/standalone_pgadmin/pod.go

```diff
@@ -83,7 +83,7 @@ func pod(
 		Env: []corev1.EnvVar{
 			{
 				Name:  "PGADMIN_SETUP_EMAIL",
-				Value: inPGAdmin.Spec.AdminUsername,
+				Value: fmt.Sprintf("admin@%s.%s.svc", inPGAdmin.Name, inPGAdmin.Namespace),
```

Comment:
is there a way to grab this whole string somewhere? Instead of generating this?

Reply:
I assume there wouldn't be. We'd have to pull it from a resource that hasn't been created yet :thinking: